code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -149,3 +149,10 @@
 	icon_state = "pill18"
 	list_reagents = list("insulin" = 50)
 	roundstart = 1
+
+/obj/item/weapon/reagent_containers/pill/mining
+	name = "Basic brute and burn pill"
+	desc = "Contains 20 units of bicaridine and kelotane, for a cheap way to mend those lacerations!"
+	icon_state = "pill20"
+	list_reagents = list("Bicaridine" = 20, "Kelotane" = 20)

```

Comment:
This is incredibly strong; it'll heal a total of 100 brute and 100 burn damage for a single pill.